platform/features/plot/src/policies/PlotViewPolicy.js

```diff
@@ -30,30 +30,25 @@ define(
          * @constructor
          * @memberof platform/features/plot
          */
-        function PlotViewPolicy() {
+        function PlotViewPolicy(openmct) {
+            this.openmct = openmct;
         }
 
-        function hasNumericTelemetry(domainObject) {
-            var telemetry = domainObject &&
-                    domainObject.getCapability('telemetry'),
-                metadata = telemetry ? telemetry.getMetadata() : {},
-                ranges = metadata.ranges || [];
-
-            // Generally, we want to allow Plot for telemetry-providing
-            // objects (most telemetry is plottable.) We only want to
-            // suppress this for telemetry which only has explicitly
-            // non-numeric values.
-            return ranges.length === 0 || ranges.some(function (range) {
-                    // Assume format is numeric if it is undefined
-                    // (numeric telemetry is the common case)
-                    return range.format === undefined ||
-                        range.format === 'number';
-                });
-        }
+        PlotViewPolicy.prototype.hasNumericTelemetry = function (domainObject) {
+            var adaptedObject = domainObject.useCapability('adapter');
+            var metadata = this.openmct.telemetry.getMetadata(adaptedObject);
+            var rangeValues = metadata.valuesForHints(['range']);
+            if (rangeValues.length === 0) {
+                return false;
+            }
+            return !rangeValues.every(function (value) {
+                return value.format === 'string';
```

Comment:
I think this is an area where our telemetry metadata is underspecified. All this check tells you is that a particularly property is not meant to be formatted with `toString`; we presume that it is plottable because we have an undocumented rule that all telemetry is numeric except for strings. Not sure what a solution should look like. A `numeric` hint would be consistent, but that would be a lot of boilerplate

Reply:
this is here solely so the event message generator doesn't show plots.  because the eventgenerator uses old telemetry metadata, it defines `ranges`.  once transitioned, it won't incorrectly define ranges.